resource_retriever/package.xml

```diff
@@ -12,8 +12,7 @@
 
   <author email="jfaust@willowgarage.com">Josh Faust</author>
   <author email="isucan@gmail.com">Ioan Sucan</author>
-  <maintainer email="clalancette@osrfoundation.org">Chris Lalancette</maintainer>
-  <maintainer email="sloretz@osrfoundation.org">Shane Loretz</maintainer>
```

Comment:
Mind moving Chris and I to `author` tags? That's the usual practice when changing maintainers.